test/simpleTestForGenerator/NTP-0.txt

```diff
@@ -0,0 +1,5 @@
+NTP No Copyright (NTP-0)
+
+Copyright (4-digit-year) by (CopyrightHoldersName)
+
+Permission to use, copy, modify, and distribute this software and its documentation for any purpose is hereby granted, provided that the name(s) of  (TrademarkedName) not be used in advertising or publicity pertaining to distribution of the software without specific, written prior permission. (TrademarkedName) make(s) no representations about the suitability of this software for any purpose. It is provided "as is" without express or implied warranty.
```

Comment:
There are two space characters in `of··(TrademarkedName)`.

Reply:
Thanks @waldyrious .  I see that now in my local copy of the .txt file.  I'll go ahead and fix that now.  Out of curiosity, how did you find that, given that the tests passed?

Reply:
I have the [Refined GitHub](https://github.com/sindresorhus/refined-github) extension installed in my browser, and one of its features is [highlighting whitespace in code and diffs](https://github.com/sindresorhus/refined-github#code), like this:

![screenshot](https://user-images.githubusercontent.com/1402241/61187598-f9118380-a6a5-11e9-985a-990a7f798805.png)

Reply:
Very nice -- thank you @waldyrious .